src/main/java/seedu/sudowudo/logic/commands/AddCommand.java

```diff
@@ -38,6 +38,8 @@
     private final Item toAdd;
     private Item toUndoAdd;
     private boolean hasTimeString = false;
+    
+    private final DateTimeParser dtparser = DateTimeParser.getInstance();
```

Comment:
Variable name seems odd.. maybe `dateTimeParser`? or `dtParser` using camelcase if you must haha